<launch>

    <!-- Actor trajectory & state publisher -->
    <node pkg="rpf_ws" type="actor_state_publisher" name="actor_state_publisher" output="screen">
        <param name="actor_name"            value="actor0"/>
        <param name="frame_id"              value="map"/>
        <param name="yaw_offset"            value="-1.5708"/>
        <param name="trajectory_duration"   value="8.0"/>
        <param name="trajectory_width"      value="0.06"/>
    </node>

    <!-- Teleop keyboard for actor -->
    <node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="actor0_teleop" output="screen" launch-prefix="xterm -e">
        <remap from="cmd_vel" to="actor0/cmd_vel"/>
    </node>

</launch>